auto-instrumentation/okhttp/okhttp-3.0/library/build.gradle.kts

```diff
@@ -3,9 +3,9 @@ plugins {
     id("otel.publish-conventions")
 }
 
-val otelVersion = project.property("otel.sdk.version")
 dependencies {
-    api("com.squareup.okhttp3:okhttp:3.0.0")
-    api("io.opentelemetry.instrumentation:opentelemetry-okhttp-3.0:$otelVersion-alpha")
-    implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api-semconv:$otelVersion-alpha")
+    implementation(platform(project(":dependencyManagement")))
+    api("com.squareup.okhttp3:okhttp")
```

Comment:
Only in this library, which will be used at runtime by a consumer project, I think we should leave the version to `3.0.0` because that's the minimum version supported by this auto instrumentation. If we set a higher version here, then we'd enforce our version into a project that might be using `3.0.0` as gradle would always pick the highest version when 2 or more are defined.